Types of Contributions

```diff
@@ -0,0 +1,264 @@
+TYPES OF CONTRIBUTIONS
+
+In this chapter, we will discuss the types of contribution that open source projects involved.
+
+Majorly, there are two types of contributions. Which are; 
+Non-coding type of contribution
+Coding type of contribution
+
+Non-Coding Type Of Contributions
+
+Non-coding contributions are valuable contributions made to open source projects that do not involve writing or modifying code. These contributions are essential for the overall health and success of open source communities and projects. They help improve documentation, user experience, project management, and community engagement.
+
+Here are some common types of non-coding contributions
+
+Documentation
+Graphic design
+Blog writing
+
+
+DOCUMENTATION
+
+Documentation is a crucial type of open source contribution that involves creating, improving, or maintaining the written resources that accompany open source software projects. Good documentation can make a project more accessible and user-friendly, and it can help attract new contributors.
+
+Contributing to documentation in open source projects is a valuable way to make a meaningful impact, even if you're not a developer. Clear and well-organized documentation enhances the overall quality of the project and fosters a more welcoming and inclusive open source community.
+
+
+Importance of good documentation
+
+Ease of Use: Well-documented open source projects are more accessible to users and developers. Clear and comprehensive documentation helps users understand how to install, configure, and use the software effectively.
+
+Onboarding: Documentation is essential for onboarding new contributors to a project. It provides them with the necessary information to get started, including project goals, coding standards, and development workflows.
+
+
+Troubleshooting: When users or developers encounter issues or bugs, documentation often contains troubleshooting guides and FAQs that can help them resolve problems without needing to contact the project maintainers.
+
+Knowledge Sharing: Documentation is a means of sharing knowledge and best practices within the open source community. It allows experienced developers to pass on their expertise to others and ensures that knowledge is not lost as contributors come and go.
+
+Community Growth: High-quality documentation can attract more users and contributors to an open source project. It builds trust and confidence in the project, which can lead to a larger and more active community.
+
+Maintainability: Documentation can also encompass guides on how to contribute to the project, including coding guidelines and codebase architecture. This helps maintain consistency and makes it easier for developers to contribute effectively.
+
+
```

Comment:
```suggestion
### How to Contribute to Documentation
If you're interested in contributing to documentation, here are a few tips:

Read the documentation: This will help you understand how the product is used.

Find a task that's a good fit for your skills and interests: If you're not sure where to start, ask the project's maintainers for suggestions.

Use the product: Gaining user insight will help you see which information is needed or not useful in the docs.  

Be patient and persistent: It takes time to write good documentation. Don't be afraid to ask for help if you get stuck.

```